src/plugins/intel_cpu/src/node.h

```diff
@@ -717,7 +717,8 @@ class Node {
     //     is still under control of strong references outside of cache.
     // privateWeightCache is for holding strong references to constant weight
     // copies of same content with different layouts.
-    std::unordered_map<std::string, MemoryPtr> privateWeightCache;
+    std::shared_ptr<std::unordered_map<std::string, MemoryPtr>> privateWeightCache
+    = std::make_shared<std::unordered_map<std::string, MemoryPtr>>();
```

Comment:
With the proposed version we never create a map object.

Reply:
The comment is only about default initialization with `nullptr` instead of default constructor of `shared_ptr`. But the usage of a `shared_ptr` is fine.